### PR TITLE
Adding Homebrew instructions

### PR DESCRIPTION
**Pull request by benfrain**
Followed https://github.com/JohnnyMorganz/StyLua/issues/237 some time ago and noticed the README never got updated. Instructions for homebrew now in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,13 @@ If you are using Docker, the easiest way to install StyLua is:
 COPY --from=JohnnyMorganz/StyLua:0.17.0 /stylua /usr/bin/stylua
 ```
 
+### Homebrew
+StyLua is available on macOS via the [Homebrew](https://brew.sh) package manager.
+
+```sh
+brew install stylelua
```
Comment (JohnnyMorganz) [inline README.md:88]:
```suggestion
brew install stylua
```

Unless ofcourse it is actually uploaded to homebrew with that name

**Review by JohnnyMorganz — COMMENTED** [review]
Awesome, thank you!

File: README.md (hunk 1 above)
Comment (benfrain) [reply README.md:88]: You are correct, apologies!

**Review by JohnnyMorganz — APPROVED** [review]
Thank you!

**Pull request merged**